core/controller/src/main/scala/whisk/core/entitlement/ActivationThrottler.scala

```diff
@@ -69,6 +69,7 @@ class ActivationThrottler(loadBalancer: LoadBalancer, concurrencyLimit: Identity
 sealed trait RateLimit {
   def ok: Boolean
   def errorMsg: String
+  val limitName: String = this.getClass.getSimpleName
```

Comment:
Please don't use `this.getClass.getSimpleName`, rather set an override for each specific limit.